test/startservers.py

```diff
@@ -93,6 +102,8 @@ def start(race_detection):
             if not check():
                 return False
             ports = range(8000, 8005) + [4000, 4430]
+            if default_config_dir.startswith("test/config-next"):
+                ports.extend([8011, 8012])
```

Comment:
Can you add a comment to link these debug port #'s to the two remote VA instances? A casual reader has to cross reference with the config files.